tests/monitoring/prometheus_utils.go

```diff
@@ -227,3 +228,9 @@ func killPortForwardCommand(portForwardCmd *exec.Cmd) error {
 	_, err := portForwardCmd.Process.Wait()
 	return err
 }
+
```

Comment:
@assafad I think you can move this function closer to the consumer (`tests/monitoring/monitoring.go`), since it isn't being re-used anywhere else except the test suite. You will also spare the import of `monitoringv1` here.

Reply:
Done